languages/de/tl_module.xlf

```diff
@@ -20,6 +20,16 @@
         <source>Please select an activation notification.</source>
         <target>Bitte wählen Sie eine Aktivierungsbenachrichtigung.</target>
         
+      </trans-unit>
+      <trans-unit id="tl_module.nc_close_notification.0">
+        <source>Close notification</source>
+        <target>Schließungsbenachrichtigung</target>
+        
+      </trans-unit>
+      <trans-unit id="tl_module.nc_close_notification.1">
+        <source>Please select a close notification.</source>
+        <target>Bitte wählen Sie eine Schließungsbenachrichtigung.</target>
+        
```

Comment:
Translations are managed on [Transifex](https://explore.transifex.com/terminal42/notification_center/) - you only need to provide the English translations and remove the other languages.